### PR TITLE
New Feedly Cheat Sheet

### PR DESCRIPTION
**Pull request by tossj**
## Description of new Instant Answer, or changes

Creates a cheat sheet for the keyboard shortcuts of the Feedly RSS reader web app.

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/feedly_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
Thanks @tossj 
However, this doesn't follow our communities guidelines. Please refer to our [contributing.md](https://duckduckhack.com/contributing.html#not-following-these-guidelines) and ping me or another community leader to reopen.

**Comment by tossj** [comment]
@pjhampton I believe I've fixed the PR comment. Would you reopen this PR?

**Comment by pjhampton** [comment]
Apologies - I should have been more clear. Please update with the [PR Template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

File: share/goodie/cheat_sheets/json/feedly.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                "key": "j",
+                "val": "Inline next story"
+            },
+            {
+                "key": "k",
+                "val": "Inline previous story"
+            },
+            {
+                "key": "n",
+                "val": "Select next story"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/feedly.json]: According to the [source](https://blog.feedly.com/feedly-keyboard-shortcuts/) - `n` should do this - `next unread article` which seems to be inconsistent with `Select next story`.

Comment (tossj) [inline share/goodie/cheat_sheets/json/feedly.json]: @sahildua2305 I've updated the source link to a more recent Feedly knowledge base article, but even this page is not entirely up to date. I've marked this page as inaccurate, so hopefully a Feedly support member will update the keyboard shortcuts.

Alternatively, I could find a source that is up to date but not officially from Feedly, but I thought an official source would be better.

The most up to date shortcuts are found in the picture here [https://imgur.com/46zpY88](https://imgur.com/46zpY88)

**Comment by daxtheduck** [comment]
@tossj this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by mintsoft** [comment]
Many Thanks @tossj

**Pull request merged**